src/runtime/entitlements-manager.js

```diff
@@ -600,6 +600,14 @@ export class EntitlementsManager {
             'crypt=' +
               encodeURIComponent(params.encryption.encryptedDocumentKey)
           );
+
+          // Add swgUserToken param.
+          if (params.encryption.swgUserToken) {
+            urlParams.push(
+              'swgUserToken=' +
```

Comment:
Is there any guidance on query parameter names like this (which hint at user identification)? Should it be more obfuscated?

Reply:
Discussed offline. We will use param 'sut'. BE will be updated too.